### ISSUE TITLE
Saved trends list

### ISSUE THREAD

**Issue by prawnsalad** [PostHog/posthog#941]
**Is your feature request related to a problem? Please describe.**
A dashboard of trends/funnels/etc is great for quick overviews but sometimes individual trends are only used once in a while and not important enough for a dashboard. They can also clog up a dashboard and slow it down.

**Describe the solution you'd like**
Just like the Funnels page, add a section of saved trends.

**Describe alternatives you've considered**

**Additional context**

***Thank*** you for your feature request - we love each and every one!

**Comment by fabrizzioalco** [comment]
Hey, can i have more context about this please

**Comment by flickz** [comment]
@timgl Can you assign me this issue?

**Comment by timgl** [comment]
Apologies, this will actually be solved once #1381 is merged so closing this for now

**Comment by flickz** [comment]
I don't think it should be closed since the PR hasn't been merged, I just think the PR should reference the issue or maybe have the issue assigned, to indicate someone is working on it.

**Comment by timgl** [comment]
@flickz This ticket was superseded by #1299 :)
